README.md

```diff
@@ -1,6 +1,6 @@
 # road2crypto-wallet-address-validator
 
-A simple utility to validate a cryptocurrency wallet addresses.
+A simple utility to validate cryptocurrency wallet addresses for EVM, Solana, and Bitcoin chains.
```

Comment:
You reverted my change, this needs to say: "A simple utility to validate a cryptocurrency wallet address."

Reply:
oops, updated